### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a contribution document and a getting started for local development.

### CODE REVIEW

File: docs/CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contribution
+
+Please review the contriibutions guidance for this project. Failure to adhere to the guidelines could result in pull requestes (PR) getting denied.
```
Comment (circa10a) [inline docs/CONTRIBUTION.md:3]: > contriibutions

small typo

File: docs/CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Contribution
+
+Please review the contriibutions guidance for this project. Failure to adhere to the guidelines could result in pull requestes (PR) getting denied. 
+
+To start contributing please open up an issue, unless there is an issue already open. Issues are used to dicuss feature requests, bug fixes, and all other changes to project.
```
Comment (circa10a) [inline docs/CONTRIBUTION.md:5]: > dicuss

This should probably read

> To start contributing, please open up an issue unless there is an issue already open

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+The bot also needs the following permissions.
+
+![image of bot permissions that are required](../public/images/bot_permissions.png)
+
+To invite Ava to a Discord test channel, use the generated URL. Add the generated URL to your browser search bar and visit the URL.
+
+![](../public/images/bot_invite_link.png)
+
+
```
Comment (circa10a) [inline docs/getting-started.md:31]: Should we remove this extra line?

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+The next page will take you to the bot channel invite page. Go ahead and select your Discord channel of where you want to test the bot.
+
+![View of Discord invite modal with channel prompt](../public/images/add_bot_channel.png)
+
+## Bot Token
+The bot token can be found on your test application's bot page. You will need the token value for the next step.
```
Comment (circa10a) [inline docs/getting-started.md:37]: Should have a new line break under the header

File: docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+![bot token view](../public/images/bot_token_step.png)
+
+
```
Comment (circa10a) [inline docs/getting-started.md:41]: unnecessary new line here

File: docs/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+> ava@1.0.0 start
+> node ./index.js
+
+[INFO] Ready!
+```
+
+### Non-Docker
+
+Ava can run locally as a nodejs process. To start Ava as a node.js process, issue the command into your terminal. You may have to issue `npm install` prior to starting the server if you have not done so prior..
```
Comment (circa10a) [inline docs/getting-started.md:69]: > nodejs process

Should be node.js process. Also have an extra period on the end here.

File: docs/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+> node ./index.js
+
+[INFO] Ready!
+```
+
+## Verify Ava
+
+Go vist your Discord test channel to verify the test bot is operational.
+
+Go ahead and issue the `ava list` command in the channel. If everything is working correctly, you should see an output containing all the available Ava commands.
```
Comment (circa10a) [inline docs/getting-started.md:89]: > you should see an output

Should probably say 

> you should see a reply

or

> you should see a message

File: docs/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+![ava returns commands](../public/images/ava_validate.png)
+
+## Closing
+
+If Ava returned all the commands then congratulations. You now have local test server, and an ava test bot to verify new commands.
```
Comment (circa10a) [inline docs/getting-started.md:95]: > You now have local test server

Should probably read

> You now have **a** local test server

**Review by circa10a — CHANGES_REQUESTED** [review]

**Comment by karl-cardenas-coding** [comment]
@circa10a Thanks for the review. Nice catch on those typos.

**Comment by circa10a** [comment]
Awesome job on the docs!

**Pull request merged**